app/src/androidTest/java/org/projectbuendia/client/ui/chart/PatientChartActivityTest.java

```diff
@@ -51,6 +58,9 @@
     private static final String YES = "●";
 
     private static final int ROW_HEIGHT = 84;
+    private static final Matcher<View> OVERFLOW_BUTTON_MATCHER = anyOf(
+            allOf(isDisplayed(), withContentDescription("More options")),
+            allOf(isDisplayed(), withClassName(endsWith("OverflowMenuButton"))));;
 
```

Comment:
I haven't seen this Android Style Guide before, just the java Style.
I've updated our wiki side bar to link to this page too.